tests/lib/azure/vnet.go

```diff
@@ -24,13 +25,21 @@ func NewVnet(f fixture.F) *Vnet {
 
 // AssertExists checks if virtual network exists in the resource group.
 // Fail tests otherwise.
-func (vnet *Vnet) AssertExists(t *testing.T, name string, address string) {
+func (vnet *Vnet) AssertExists(t *testing.T, name string, address, routeTable string) {
 	vnet.f.Retrier.Run(newID("Vnet", "AssertExists", name), func() error {
 		net, err := vnet.client.Get(vnet.f.ResGroupName, name, "")
 		if err != nil {
 			return err
 		}
 
+		if net.VirtualNetworkPropertiesFormat == nil {
+			return errors.New("The field VirtualNetworkPropertiesFormat is nil!")
+		}
+		properties := *net.VirtualNetworkPropertiesFormat
+
+		if properties.AddressSpace == nil {
+			return errors.New("The field AddressSpace is nil!")
+		}
 		addressActual := *net.VirtualNetworkPropertiesFormat.AddressSpace.AddressPrefixes
```

Comment:
A simpler name that I found on the gopl book is use the pattern name "got" for variables that are what your code produced. The "expected" values they call "want". So the final code is:

```
if got != want {
// etc
}
```

I liked the got, but want...it seems a little odd and I like expected more...but want is smaller.